### ISSUE TITLE
[displacy] Dependency problem

### ISSUE THREAD

**Issue by EtienneAb3d** [explosion/spaCy#3725]
When installing with the official script, `app.py` is not working properly. The solution is to provide with exact versions, like this:
```
hug==2.4.8
hug-middleware-cors==1.0.0
spacy==2.1.0
waitress==1.3.0
```

**Comment by ines** [comment]
Which `app.py` are you referring to? The displaCy service from [`spacy-services`](https://github.com/explosion/spacy-services/tree/master/displacy)?

There has been a recent update to Hug that seems to have broken things due to dependency conflicts. We're always trying to pin dependencies in a sensible way (allow minor updates and patches, but prevent automatically upgrading to new major versions). But it sometimes still happens that updates to the dependencies break things.

**Comment by EtienneAb3d** [comment]
Yes :)

**Comment by ines** [comment]
Okay, I included the Hug version pin in the v2.1 update branch and merged that: https://github.com/explosion/spacy-services/pull/20
